apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -161,15 +162,23 @@ export class DsbMessagePoolingService implements OnModuleInit {
   }
 
   public async sendMessagesToSubscribers(messages: GetMessageResponse[], fqcn: string, client: any): Promise<void> {
-    const emitMode: EventEmitMode = this.configService.get('EVENTS_EMIT_MODE', EventEmitMode.BULK);
+    try {
+      const emitMode: EventEmitMode = this.configService.get('EVENTS_EMIT_MODE', EventEmitMode.BULK);
 
-    if (emitMode === EventEmitMode.BULK) {
-      client.send(JSON.stringify(messages.map((message) => ({ ...message, fqcn }))));
-      return;
-    }
+      if (emitMode === EventEmitMode.BULK) {
+        const msg = JSON.stringify(messages.map((message) => ({ ...message, fqcn })));
+        this.logger.log(`[WS][sendMessagesToSubscribers][BULK]${msg}`);
+        client.send(msg);
+        return;
+      }
+
+      messages.forEach((message: GetMessageResponse) => {
+        this.logger.log(`[WS][sendMessagesToSubscribers][SINGLE]${message}`);
+        client.send(JSON.stringify({ ...message, fqcn }));
+      });
 
-    messages.forEach((message: GetMessageResponse) => {
-      client.send(JSON.stringify({ ...message, fqcn }));
```

Comment:
move printing error object outside of this string, in some cases it may not be parsed properly/not contain everything